src/Service/TokenService.php

```
@@ -0,0 +1,229 @@
+    {
+        return $this->refreshTokenRepository->revokeAllForUserExcept($user, $except);
+    }
+
+    /**
+     * Gets the private key for signing tokens.
+     */
+    private function getPrivateKey(): \OpenSSLAsymmetricKey
+    {
+        $keyContents = file_get_contents($this->privateKeyPath);
+        if (false === $keyContents) {
+            throw new \RuntimeException('Failed to read private key file');
```

PHPMD (2.15.0)

Missing class import via use statement (line '201', column '23'). (undefined)

(MissingImport)